bin/ashley

```diff
@@ -3,4 +3,6 @@
 declare DOCKER_USER
 DOCKER_USER="$(id -u):$(id -g)"
 
-DOCKER_USER=${DOCKER_USER} docker-compose run --service-ports ashley
+DOCKER_USER=${DOCKER_USER} docker-compose up -d postgresql
```

Comment:
I didn't notice this file before. In our project all the run operations are made using a `Makefile`. basically your makefile should have this task `bootstrap`, `build`, `run`, `stop`, `destroy`. But this out of the scope of this PR.

Reply:
Thanks, I take note and i'll address this in another PR.